### ISSUE TITLE
Query parameters on Submissions index page do not persist on page reload

### ISSUE THREAD

**Issue by chrismlusk** [verbb/formie#1727]
### Describe the bug

When on the Submissions index page, if you reload the page after changing some of the toolbar options, some of the query parameters do not persist — so the filtered view is lost. Aside from reloading the page, this makes it so it is not possible to share URLs to this page with a specific state.

### Steps to reproduce

1. Go to the Submissions index page
2. Change the Status from All → Spam
3. Reload the page

### Craft CMS version

4.7.2.1

### Plugin version

2.1.3

**Comment by chrismlusk** [comment]
Actually, I'm seeing different quirks with the status menu, search, and date range menu.

### Status Menu

See above.

### Search 

Type something in the input and then reload. The `search` query parameter is lost — but the value is still in the input.

### Date range menu

No query parameter is set, and the selected option does not persist on reload.

---

I'm not sure if these are bugs or if this is now a feature request! So, my ask is if these element index pages can work like core Craft ones.

**Comment by engram-design** [comment]
Should be fixed for status, and the search text seems to be working for me. The date ranges are a little thicker, as that's a Craft `createDateRangePicker` control, which values are only the actual start/end date values, which isn't really helpful if you want to share "Today's" option with people, as that'll include the timestamps in the URL, not just `?range=today` for example.

**Comment by chrismlusk** [comment]
To clarify on the search text, sharing a URL works as expected. But after a page reload, the query parameter is lost from the URL — although the input retains the value. So if you reload a second time, the search text gone completely from both the URL and the input.

**Comment by engram-design** [comment]
Yeah, I thought that might be what you meant, but the query param does persist on my end after refreshes, and opening via the link in a new tab. I'll keep at trying to replicate.
